src/components/app-cta/app-cta.jsx

```diff
@@ -29,6 +30,9 @@ var AppCta = React.createClass({
     });
   },
   openApp: function () {
+
+    platform.trackEvent('Browser Player', 'Click CTA', 'CTA: Open app', this.getDeepLink());
```

Comment:
The fourth input in [this function](https://github.com/mozilla/webmaker-core/blob/develop/src/lib/platform.js#L167) ( `value` ) only accepts an integer in GA.

So I suggest we do this instead: 


```js
platform.trackEvent('Browser Player', 'Click CTA Open App', this.getDeepLink());
```